public/src/components/channelManagement/MutliSelectTagEditor.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import { makeStyles } from '@mui/styles';
+import { grey } from '@mui/material/colors';
+import { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  container: {
+    width: '100%',
+    '& > * + *': {
+      marginTop: spacing(3),
+    },
+    borderColor: `2px solid ${{ color: grey[700] }}`,
+    borderRadius: '2px',
+    padding: spacing(2),
+  },
+}));
+
+interface Option {
+  label: string;
+  value: string;
+}
+
+const options: Option[] = [
+  { label: 'environment/climate-change', value: 'environment/climate-change' },
+  { label: 'environment/climate-crisis', value: 'environment/climate-crisis' },
+  { label: 'environment/environment', value: 'environment/environment' },
+  { label: 'science/science', value: 'science/science' },
+  { label: 'politics/politics', value: 'politics/politics' },
+  { label: 'us-news/us-politics', value: 'us-news/us-politics' },
+  { label: 'australia-news/australian-politics', value: 'australia-news/australian-politics' },
+  { label: 'world/world', value: 'world/world' },
+  { label: 'world/europe-news', value: 'world/europe-news' },
+  { label: 'world/russia', value: 'world/russia' },
+  { label: 'books/books', value: 'books/books' },
+  { label: 'culture/culture', value: 'culture/culture' },
+  { label: 'world/coronavirus-outbreak', value: 'world/coronavirus-outbreak' },
+  { label: 'world/race', value: 'world/race' },
+  { label: 'inequality/inequality', value: 'inequality/inequality' },
+  { label: 'technology/technology', value: 'technology/technology' },
+  { label: 'business/business', value: 'business/business' },
+  { label: 'tone/recipes', value: 'tone/recipes' },
+];
+
+interface MultiselectAutocompleteProps {
+  disabled: boolean;
+  tags: string[] | null;
+  onUpdate: (tags: string[]) => void;
+}
+
+const MultiselectAutocomplete: React.FC<MultiselectAutocompleteProps> = ({
+  disabled,
+  tags,
+  onUpdate,
+}: MultiselectAutocompleteProps) => {
+  const classes = useStyles();
+
+  const [inputValue, setInputValue] = React.useState<string>('');
+
+  return (
+    <div className={classes.container}>
+      <span style={{ fontSize: '1rem', fontWeight: 'normal' }}>Tag for content tagged article</span>
```

Comment:
```suggestion
      <span style={{ fontSize: '1rem', fontWeight: 'normal' }}>Tags for content tagged article</span>
```
Plural "Tags" since it can be multiple maybe